easybuild/easyconfigs/i/intel/intel-2016.02.eb

```diff
@@ -0,0 +1,25 @@
+# This is an easyconfig file for EasyBuild, see http://hpcugent.github.io/easybuild
+easyblock = "Toolchain"
+
+name = 'intel'
+version = '2016.02'
```

Comment:
good and useful PR, nice to see it merged.

As said during meeting today: it would be good to have a `-GCC-5.3.0` suffix, to make room for `-GCC-4.9.3`